packages/venia-ui/lib/components/Gallery/addToCartButton.js

```diff
@@ -28,15 +44,26 @@ const AddToCartButton = props => {
         />
     );
 
+    const buttonAriaLabel = formatMessage({
```

Comment:
could combine all the in/ out of stock to one object so it will be easier to keep track of state changes.

Reply:
ok